docs/user/tutorials/01-30-azure-vpc-peering.md

```diff
@@ -8,178 +8,173 @@ the steps that create those resources.
-               memory: 256Mi
-               cpu: "0.2"
-           image: ubuntu
-           command:
-             - "/bin/bash"
-             - "-c"
-             - "--"
-           args:
-             - "apt update; apt install iputils-ping -y; ping -c 20 $IP_ADDRESS"
-   EOF
-   ```
-
-   This workload should print a sequence of 20 echo replies to stdout.
-
-12. Print the logs of one of the workloads, run:
-
-   ```shell
-   kubectl logs -n $NAMESPACE `kubectl get pod -n $NAMESPACE -l app=azurevpcpeering-demo -o=jsonpath='{.items[0].metadata.name}'`
-   ```
-
-   The command should print something like:
-   ```
-   ...
-   PING 172.0.0.4 (172.0.0.4) 56(84) bytes of data.
-   64 bytes from 172.0.0.4: icmp_seq=1 ttl=63 time=8.10 ms
-   64 bytes from 172.0.0.4: icmp_seq=2 ttl=63 time=2.01 ms
-   64 bytes from 172.0.0.4: icmp_seq=3 ttl=63 time=7.02 ms
-   64 bytes from 172.0.0.4: icmp_seq=4 ttl=63 time=1.87 ms
-   64 bytes from 172.0.0.4: icmp_seq=5 ttl=63 time=1.89 ms
-   64 bytes from 172.0.0.4: icmp_seq=6 ttl=63 time=4.75 ms
-   64 bytes from 172.0.0.4: icmp_seq=7 ttl=63 time=2.01 ms
-   64 bytes from 172.0.0.4: icmp_seq=8 ttl=63 time=4.26 ms
-   64 bytes from 172.0.0.4: icmp_seq=9 ttl=63 time=1.89 ms
-   64 bytes from 172.0.0.4: icmp_seq=10 ttl=63 time=2.08 ms
-   64 bytes from 172.0.0.4: icmp_seq=11 ttl=63 time=2.01 ms
-   64 bytes from 172.0.0.4: icmp_seq=12 ttl=63 time=2.24 ms
-   64 bytes from 172.0.0.4: icmp_seq=13 ttl=63 time=1.80 ms
-   64 bytes from 172.0.0.4: icmp_seq=14 ttl=63 time=4.32 ms
-   64 bytes from 172.0.0.4: icmp_seq=15 ttl=63 time=2.03 ms
-   64 bytes from 172.0.0.4: icmp_seq=16 ttl=63 time=2.03 ms
-   64 bytes from 172.0.0.4: icmp_seq=17 ttl=63 time=5.19 ms
-   64 bytes from 172.0.0.4: icmp_seq=18 ttl=63 time=1.86 ms
-   64 bytes from 172.0.0.4: icmp_seq=19 ttl=63 time=1.92 ms
-   64 bytes from 172.0.0.4: icmp_seq=20 ttl=63 time=1.92 ms
-   
-   === 172.0.0.4 ping statistics ===
-   20 packets transmitted, 20 received, 0% packet loss, time 19024ms
-   rtt min/avg/max/mdev = 1.800/3.060/8.096/1.847 ms
-   ...
-   ```
-
+    ```shell
+    kubectl apply -n $NAMESPACE -f - <<EOF
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: azurevpcpeering-demo
+    spec:
+      selector:
+        matchLabels:
+          app: azurevpcpeering-demo
+      template:
+        metadata:
+          labels:
+            app: azurevpcpeering-demo
+        spec:
+          containers:
+          - name: my-container
+            resources:
+              limits:
+                memory: 512Mi
+                cpu: "1"
+              requests:
+                memory: 256Mi
+                cpu: "0.2"
+            image: ubuntu
+            command:
+              - "/bin/bash"
+              - "-c"
+              - "--"
+            args:
+              - "apt update; apt install iputils-ping -y; ping -c 20 $IP_ADDRESS"
+    EOF
+    ```
+
+    This workload should print a sequence of 20 echo replies to stdout.
```

Comment:
Why "should"? Is this uncertain?
Why not "This workload prints a sequence of 20 echo replies to stdout."
Did something go wrong if it doesn't print 20 echo replies to stdout?

Reply:
It is more like if VPC peering is in place this command will print a sequence of 20 echo replies to stdout.

I agree that it is nicer without should.